src/components/Updated/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import moment from 'moment';
```

Comment:
is moment an alternative to datejs?

Reply:
I think we forgot about datejs, so came across this instead. It seems to do the same things as datejs